scico/linop/xray/astra.py

```diff
@@ -577,8 +591,10 @@ def __init__(
             det_spacing: Spacing between detector elements. See the
                `astra documentation <https://www.astra-toolbox.com/docs/geom3d.html#projection-geometries>`__
                for more information.
-            angles: Array of projection angles in radians.
-            vectors: Array of geometry specification vectors.
+            angles: Array of projection angles in radians, mutually
+                exclusive with `vectors`.
```

Comment:
Perhaps "... radians. This parameter is mutually exclusive with ...", and likewise for the line below?

Also, before "Args:", "These options are ..." would be better as "These parameters are ..."